config/config.go

```diff
@@ -106,6 +106,13 @@ func (a *Auth) Obfuscate() {
 	a.CAFile = "xxx"
 }
 
+// Thanos describes configuration of the Thanos proxy component
+type Thanos struct {
```

Comment:
shouldn't we call this ThanosProxy just to remain consistent with the json name?